packages/p2p/src/stores/general-store.js

```diff
@@ -353,6 +354,11 @@ export default class GeneralStore extends BaseStore {
         this.is_restricted = is_restricted;
     }
 
+    @action.bound
+    setModalOpenState(is_modal_open) {
```

Comment:
Could we maybe use something like `setIsModalOpen` instead?

Reply:
Will rename